### PR TITLE
FBFW-127 - Get all Messages in Redux

### PR DESCRIPTION
**Pull request by mariomelchor**
Added a new allMessages parameter when passed in loads all messages 

- Confirm all messages appear in redux state when game loads
- Confirm no messages are seen when you don't have authentication
- Confirm no messages returned For No `Press Type =  Press` or `Press Type =  Rulebook Press During Builds/Retreats `

Screen Shot 2022-05-07 at 10 52 35 PM

Screen Shot 2022-05-07 at 10 50 24 PM

#### http://localhost/api.php?route=game/getmessages&gameID=5&countryID=4&allMessages

Screen Shot 2022-05-07 at 10 51 06 PM

## For No Press or During Builds/Retreats for Rulebook Press 

Screen Shot 2022-05-07 at 10 56 24 PM
Screen Shot 2022-05-07 at 10 54 55 PM

## No Authentication
Screen Shot 2022-05-07 at 10 59 34 PM

### CODE REVIEW

File: beta-src/src/App.tsx [hunk 1]
```diff
@@ -12,11 +12,15 @@ const App: React.FC = function (): React.ReactElement {
...
   const dispatch = useAppDispatch();
+  const countryID = useAppSelector((state) =>
+    String(state.game.overview.user.member.countryID),
+  );
   dispatch(fetchGameOverview({ gameID: currentGameID as string }));
   dispatch(
     fetchGameMessages({
       gameID: currentGameID as string,
-      countryID: "0" as string,
+      countryID,
```
Comment (bryancodazen) [inline beta-src/src/App.tsx:22]: can we just do countryID as string here to keep it consistent with gameID? I think it is preferable than type casting it as we're fetching it from global state.

**Review by bryancodazen — APPROVED** [review]
LGTM

File: beta-src/src/App.tsx (hunk 1 above)
Comment (mariomelchor) [reply beta-src/src/App.tsx:22]: @bryancodazen I initially tried that but got a type error 
Screen Shot 2022-05-09 at 9 26 47 PM

I would have to do the following.
`countryID: countryID as unknown as string,`

**Review by sariacodazen — APPROVED** [review]
LGTM

**Review by justinyoocodazen — APPROVED** [review]

**Pull request merged**